cmd/server/command/start.go

```diff
@@ -144,8 +145,12 @@ func NewStart(startOptions *startOptions) *cobra.Command {
 					}
 				}
 			}
+
+			copier := copy.New(log.With("system", "copier"))
```

Comment:
If you want to ensure that the log system is set every time, then maybe move the call inside the constructor function?

Reply:
Yeah I thought we could give this some thought. I am preparing a more general PR that injects the logger everywhere and thus it becomes very clear where this might fit.

The PR just got out of hand, so I extracted some bits eg. into this change.